x/concentrated-liquidity/incentives.go

```diff
@@ -900,6 +902,8 @@ func (k Keeper) claimAllIncentivesForPosition(ctx sdk.Context, positionId uint64
 
 	// Compute the age of the position.
 	positionAge := ctx.BlockTime().Sub(position.JoinTime)
+	// @Dev: This check seems concerning if its ever hit? Like panic worthy, how would you get a position in state from the future
+	// Do we actually need to be checking this?
 	if positionAge < 0 {
 		return sdk.Coins{}, sdk.Coins{}, types.NegativeDurationError{Duration: positionAge}
 	}
```

Comment:
I don't see why we would want to remove it. If it does occur for any unknown reason, we ar least would have a descriptive error to debug